about.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../css/about.css">
+    <script src="javascript/script.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet"> 
+</head>
+<body>
+
+    <section class="social-icons-section">
+        <ul class="social-icons">
+            <li class="network-icons-list"><a href="https://twitter.com/mkaya133" target="_blank"><img src="img/twitter.svg" alt=""></a></li>
+            <li class="network-icons-list"><a href="https://www.linkedin.com/in/mert-kaya-0732b717b/" target="_blank"><img src="img/linkedin.svg" alt=""></a></li>
+            <li class="network-icons-list"><a href="https://github.com/mkaya13" target="_blank"><img src="img/github.svg" alt=""></a></li>
+            <li class="social-icon">English</li>
+        </ul>
+    </section>
+
+    <section class="main-page">
+
+        <section class=""></section>
+
+        <nav>
+            <a href="#" class="toggle-button">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </a>
+    
+            <a href="/index.html"><img class="main-page-logo" src="img/smartbuild.jpg" alt=""></a>
+    
+            <div class="navbar-content">
+                <ul class="navbar-links">
+                    <li class="desktop-nav-elements"><a class="desktop-nav-anchors" href="#about-me">About</a></li>
+                    <li class="desktop-nav-elements"><a class="desktop-nav-anchors" href="#portfolio">Program</a></li>
+                    <li class="desktop-nav-elements"><a class="desktop-nav-anchors" href="#contact">Join</a></li>
+                    <li class="desktop-nav-elements"><a class="desktop-nav-anchors" href="#interests">Sponsor</a></li>
+                    <li class="desktop-nav-elements"><a class="desktop-nav-anchors" href="#interests">News</a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] Please make sure that the About page has a link to the Home page, one suggestion is that you change the About link to the Home link.